lib/utilities/monitoringHandler.js

```diff
@@ -167,7 +179,7 @@ function monitoringHandler(clientIP, req, res, log) {
     if (req.method !== 'GET') {
         return monitoringEndHandler(res, errors.MethodNotAllowed);
     }
-    const monitoring = (req.url === '/_/monitoring/metrics');
+    const monitoring = (req.url === '/_/metrics');
```

Comment:
small change: 
the constant defined here can be used.
https://github.com/scality/Arsenal/blob/development/8.1/lib/network/probe/ProbeServer.js#L7

Reply:
This is pending discussion on Confluence: https://scality.atlassian.net/wiki/spaces/OS/pages/1819148292/Strategy+for+exposing+metrics (where we consider using a simpler `/metrics` path, since we are already moved to a different port)

Also, importing from Arsenal does not seem look so appropriate with the current Arsenal implementation : this constant is present indeed, but deep inside the code of ProbeServer, and looks more like an implementation detail than an established constant we should rely on.

Reply:
The constant was meant to be relied upon regardless of usage to probe server, as probe server itself is optional for metrics/probes as your service may already have an established server. It is of course not "established" because it is new and only used a couple of times in backbeat as that is where it was initialized built for.

Reply:
in that case, should we not move the constant in arsenal's `constants` ?

Reply:
Uh maybe? Our use of constant.js files is a little confusing to me and I tend to go with more contextual constants rather than one mega file with everything under the sun. If you were discouraged to use the value because it was not placed in a constants file I guess we should move it. I got discouraged when I would use backbeat and there are multiple constants ( one in lib/ and one in each extension ).